configure.ac

```diff
@@ -76,6 +86,16 @@ AC_PROG_RANLIB
 AM_PROG_CC_C_O
 AC_PATH_PROG([XMLTO], [xmlto], [no])
 
+dnl elektra
+ifdef([PKG_CHECK_MODULES],
+		[PKG_CHECK_MODULES([ELEKTRA], [elektra-codegen], [ELEKTRA_LIBDIR=`pkg-config --variable=libdir elektra-codegen`],)],
+		[AC_MSG_ERROR([pkg-config not (fully) installed, but needed to find Elektra])])
+AC_SUBST([ELEKTRA_CFLAGS])
+AC_SUBST([ELEKTRA_LIBS])
+
+CFLAGS="$ELEKTRA_CFLAGS $CFLAGS"
+LIBS="$ELEKTRA_LIBS $LIBS -Wl,-rpath $ELEKTRA_LIBDIR"
+
```

Comment:
(does not belong to this line) Why is  lcdd_spec.ini and lcdexec_spec.ini binary?

Reply:
Those files shouldn't even exist. I need to look into it, but mounting the specification creates those files. I think is a result of using `specload` and `noresolver` together.

(The files should either be empty or contain only the `quickdump` magicnumber.)